src/sinks/clickhouse.rs

```diff
@@ -36,13 +37,15 @@ pub struct ClickhouseConfig {
     pub request_rate_limit_num: Option<u64>,
     pub request_retry_attempts: Option<usize>,
     pub request_retry_backoff_secs: Option<u64>,
+
+    pub basic_auth: Option<ClickHouseBasicAuthConfig>,
```

Comment:
minor nit: can we move this struct item to live under `compression`?

Reply:
@LucioFranco can you resolve this so we can merge?

Reply:
Not blocking we can merge anyways